### DISCUSSION TITLE
NBMiner v39.4

### DISCUSSION THREAD

**Discussion by nebutech-admin** [NebuTech/NBMiner#466] [General]
- `fix`: `octopus` Fix error hash issue.

- `修复`: `octopus` 修复错误提交的问题

<hr /><em>This discussion was created from the release <a href='https://github.com/NebuTech/NBMiner/releases/tag/v39.4'>NBMiner v39.4</a>.</em>

**Comment by banekondic1996** [comment]
Please make option to lock lhr value. Because GPU works best at 100 for me and i can't lock the value of it, it's just decreasing by 0.1. Can you add that option?
